recipes/TIES20/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TIES20" %}
+{% set version = "20.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCL-CCS/TIES20/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4af8f09bc480467ef8941dbc4f1a96edadb1dad6111929fc1b664eb67d266ef7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - ties = ties:cli.command_line_script
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python >=3.7
+    - pip
+    - numpy
+    - cython
+  run:
+    - python >=3.7
+    - ambertools >=20.0,<=22.0
+    - libgfortran
```

Comment:
libgfortran should not be manually added in the run section. If you need to link to a fortran library you probably should have had the fortran compiler in the build requirements.

Reply:
We can remove it. I believe we were fixing an issue in a dependency which is not relevant anymore. Thanks